diff --git a/dist/esm/hunspellLoader.js b/dist/esm/hunspellLoader.js
index a95a3d39912df63dd8bfdaf531b2b5997d82efa2..b1492016d2e941643678143d8877d1a0490685a2 100644
--- a/dist/esm/hunspellLoader.js
+++ b/dist/esm/hunspellLoader.js
@@ -1,5 +1,5 @@
 import { mountBuffer, unmount } from 'emscripten-wasm-loader';
-import * as nanoid from 'nanoid';
+import nanoid from 'nanoid';
 import { log } from './util/logger';
 import { wrapHunspellInterface } from './wrapHunspellInterface';
 /**
diff --git a/dist/esm/loadModule.js b/dist/esm/loadModule.js
index 0bb04bb9be8d45c3f6351728e51ec9c5ee7a7364..4e8a7d7505ab0e28ba973f2d71a42422eff0327e 100644
--- a/dist/esm/loadModule.js
+++ b/dist/esm/loadModule.js
@@ -2,7 +2,7 @@ import { getModuleLoader } from 'emscripten-wasm-loader';
 import { hunspellLoader } from './hunspellLoader';
 import { log } from './util/logger';
 //imports MODULARIZED emscripten preamble
-import * as runtime from './lib/node/hunspell';
+import runtime from './lib/node/hunspell';
 /**
  * Load, initialize wasm binary to use actual hunspell wasm instances.
  *
